staging/src/k8s.io/apiserver/pkg/util/flowcontrol/fairqueuing/interface.go

```diff
@@ -26,7 +28,8 @@ import (
 // each QueueSet is known to the factory, the client does not need to
 // pass this stuff in each call to NewQueueSet.
 type QueueSetFactory interface {
-	NewQueueSet(concurrencyLimit, numQueues, queueLengthLimit int, requestWaitLimit time.Duration) QueueSet
+	NewQueueSet(concurrencyLimit, numQueues, queueLengthLimit int, requestWaitLimit time.Duration, clk clock.Clock) QueueSet
```

Comment:
Keep the clock out of here.  It belongs inside the factory, as explained in the comment above.

Reply:
removed

Reply:
Did you push that change?  I do not see it.

Reply:
Sorry, it is pushed now.  The clock and wg come from the factory.